packages/ui/src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,207 @@
+import {
```

Comment:
Please don't forget to add `use client` directive for components that are using state, hooks or event handlers